src/compiler/commandLineParser.ts

```diff
@@ -244,7 +244,27 @@ namespace ts {
                 "classic": ModuleResolutionKind.Classic
             },
             description: Diagnostics.Specifies_module_resolution_strategy_Colon_node_Node_or_classic_TypeScript_pre_1_6
-        }        
+        },
+        {
+            name: "noUnusedLabels",
+            type: "boolean",
+            description: Diagnostics.Report_error_on_unused_labels
+        },
+        {
+            name: "noImplicitReturns",
+            type: "boolean",
+            description: Diagnostics.Report_error_when_not_all_code_paths_in_function_return_a_value

```

Comment:
@yuit's comment was about singular vs plural, rather than the mood, if I understood correctly. The mood inconsistency is a different issue.